evalml/pipelines/time_series_classification_pipelines.py

```diff
@@ -208,15 +218,18 @@ def _select_y_pred_for_score(self, X, y, y_pred, y_pred_proba, objective):
     def predict_in_sample(self, X, y, X_train, y_train, objective=None):
         """Predict on future data where the target is known, e.g. cross validation.
 
-        Arguments:
-            X_holdout (pd.DataFrame or np.ndarray): Future data of shape [n_samples, n_features].
-            y_holdout (pd.Series, np.ndarray): Future target of shape [n_samples].
-            X_train (pd.DataFrame, np.ndarray): Data the pipeline was trained on of shape [n_samples_train, n_feautures].
-            y_train (pd.Series, np.ndarray): Targets used to train the pipeline of shape [n_samples_train].
-            objective (ObjectiveBase, str, None): Objective used to threshold predicted probabilities, optional.
+        Args:
+            X (pd.DataFrame): Future data of shape [n_samples, n_features].
+            y (pd.Series): Future target of shape [n_samples].
+            X_train (pd.DataFrame): Data the pipeline was trained on of shape [n_samples_train, n_feautures].
+            y_train (pd.Series): Targets used to train the pipeline of shape [n_samples_train].
+            objective (ObjectiveBase, str): Objective used to threshold predicted probabilities, optional.
```

Comment:
Why did you get rid of `None`? The parameter can be None in this method.

Reply:
This could be a stylistic thing but following [Google's example](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), it seemed like even if None was an acceptable parameter, it wasn't listed as one. Instead, they used "optional" and the "Defaults as None" to indicate that!